pkgs/test/haskell/setBuildTarget/haskell-setBuildTarget.cabal

```diff
@@ -14,3 +14,8 @@ executable bar
   main-is:             Bar.hs
   build-depends:       base
   default-language:    Haskell2010
+
+test-suite tests
+  main-is:             Test.hs
+  build-depends:       base
+  default-language:    Haskell2010
```

Comment:
How about including some sort of breakage in this test, so that if it were to be compiled, it would fail?

That way we would be able to test that the test suite is actually disabled, as expected.  And GHC never even tries to compile it.